src/styles/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+  }
+
+  &__star:not(:last-of-type) {
+    margin-right: 4px;
+  }
+
+  &--4 :nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
need to move all modificators from stars task

Reply:
works incorrectly. Modifier should show how many stars are active
<img width="1222" alt="Screenshot 2024-02-22 at 17 02 21" src="https://github.com/mate-academy/layout_product-cards/assets/117587463/de1cd3bf-b16a-4140-8a0e-72e136751393">